edx_lint/pylint/module_trace.py

```diff
@@ -41,6 +41,7 @@ class ModuleTracingChecker(BaseChecker):
 
     def visit_module(self, node):
         """Called for each module being examined."""
+        self    # pylint: disable=pointless-statement
```

Comment:
Is there a reason this pointless statement is being left here? Can that reason be left in a comment next to this?

Reply:
@nedbat did this to silence the R201 warning, see https://github.com/openedx/edx-lint/pull/283